ONSdigital/response-operations-ui
# RAS-1147 Upgrade to flask-session 0.8.0 update in rOps

# What and why?
Bumping `flask-session` to version latest version which had breaking changes (for 0.8.0) to how it handled datetime objects.  With the updated code from [RAS-1177](https://jira.ons.gov.uk/browse/RAS-1177), the latest version can now be used. 

# How to test?
- Deploy this PR to your environment
- sign into rOps with the default `uaa_user` account
- check that you can send a new message
- click around making sure no 500 are raised because of a comparison error between str and datetime objects

# Jira
[RAS-1147](https://jira.ons.gov.uk/browse/RAS-1147)
